### DISCUSSION TITLE
RAG Error on reading documents

### DISCUSSION THREAD

**Discussion by rfernandez760** [open-webui/open-webui#2351] [General]
# Bug Report

## Description

**Bug Summary:**
When attaching a document for RAG a number of errors will come up.

**Steps to Reproduce:**
Attach the document used which can recreate the error (ozempic.pdf).

**Expected Behavior:**
Document comprehension.

**Actual Behavior:**
Errors get prompted

## Environment

- **Open WebUI Version:** [e.g., 0.1.120]
 v0.1.124

- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]
0.1.38

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
Ubuntu

- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]
Edge and Crhome

## Reproduction Details

**Confirmation:**

- [ x] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [ x] I have included the browser console logs.
- [ x] I have included the Docker container logs.

## Logs and Screenshots
[ozempic.pdf](https://github.com/open-webui/open-webui/files/15355235/ozempic.pdf)
[Console Log.log](https://github.com/open-webui/open-webui/files/15355280/Console.Log.log)
[Docker Log.txt](https://github.com/open-webui/open-webui/files/15355282/Docker.Log.txt)

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]
![image](https://github.com/open-webui/open-webui/assets/76540757/df64156e-3e78-4e16-831e-f47941d195e5)
![image](https://github.com/open-webui/open-webui/assets/76540757/d8b7f8fb-1559-4088-8c34-32f34955e7b4)

## Installation Method
Manual install using docker run command.

## Additional Information
Open-Webui is in a docker container, while Ollama is installed locally in Ubuntu.

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
![image](https://github.com/open-webui/open-webui/assets/25473318/0168aee6-dedd-42ee-931e-bb522c89b4a4)

**Reply by tjbck** [reply]
Have you changed the embedding model? You might want to try resetting vector db.

**Reply by rfernandez760** [reply]
Thanks Tim!  I did change the embedding model as well as the reranking. Even after the update I did reset the vector db, but doing that again seems to have resolved the issue.  On a related note, not sure if you want me to open a new ticket related to OCR scanning, it gives the following when attaching multiple files of the same type.
![image](https://github.com/open-webui/open-webui/assets/76540757/5a1c26fb-1784-4f44-8ff5-460a17adb1f7)
 
 At the least, I'll update the post to the troubleshooting area.
